locations/spiders/grimaldis_pizzeria.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from collections import namedtuple
+from locations.items import GeojsonPointItem
+
+
+class GrimaldisPizzeriaSpider(scrapy.Spider):
+    name = 'grimaldis_pizzeria'
+    allowed_domains = ['www.grimaldispizzeria.com']
+    start_urls = (
+        'https://www.grimaldispizzeria.com/locations/',
+    )
+
+    def parse(self, response):
+        locations = response.css('div.location_block')
+
+        for loc in locations:
+            address = self._format_address(loc.css('.store_address::text').getall())
+
+            properties = {
+                "ref": loc.css('h3.loc_title::text').get(),
+                "name": loc.css('.loc_title::text').get(),
+                "addr_full": address.street_address,
```

Comment:
This should be "street_address", not "addr_full"

Reply:
I get a Key error when I try to use `street_address` for the key value. 
`KeyError: 'GeojsonPointItem does not support field: street_address'`

Looks like the allowed keys for **GeojsonPointItem** are:
*`addr_full`
* `street` (I assume this must be used in combination with `housenumber`)

Reply:
Make sure your fork is up to date. We added the street_address field a week ago, and it looks like your fork is missing that change.

Reply:
Yup, that was the issue. Fixed! Though it looks like updating the fork brought in a lot of extra change into the MR. Let me know if you'd like me to try to fix that somehow.